documentation/practices.md

```diff
@@ -52,9 +39,28 @@ If you use another CMS, you must update the configurations to reflect the public
+IBM Watson Content Hub is a cloud-based CMS with an embedded content delivery network that lets marketers update content faster. It also provides services with IBM Watson, such as cognitive tagging, to help transform assets into a searchable library of content.
+
+Catalog images and site content images can be uploaded to WCH using the wchtools command-line utility. Uploaded assets are scanned using Watson's Visual Image recognition service, and tagged based on content to create a searchable library. After the images are uploaded, the content delivery URL must be provided in the Reference Storefront's configuration for both the `skuImagesUrl` and `siteImagesUrl` with the appropriate sku/file-name placeholders. For example, certain fields in the following examples are populated based on the tenant ID of the WCH account being used, and the directory structure of the assets uploaded.
 `"skuImagesUrl": "https://my11.digitalexperience.ibm.com/<wch_tenant_identifier>/dxdam/<catalog_directory>/%sku%.jpeg",`
 `"siteImagesUrl": "https://my11.digitalexperience.ibm.com/<wch_tenant_identifier>/dxdam/<site_images>/%fileName%",`
 
+## Analytics Example
+
+**Google Analytics Integration**
+
+The React PWA Reference Storefront is pre-configured for integration with the Google Analytics enhanced e-commerce plugin. 
+
+### Route/Page Views
+The handlers for integrating with Google Analytics allow for routes, which are pre-configured in the `App.js` file, to be logged as pageviews in the sites real-time traffic data. Further configuration in Google Analytics will enable measurements for conversion rates based on the pages viewed in the web traffic. 
+`Analytics.js` contains the functions used for logging page views with ReactGA (React library for invoking Google Analytics functions), and functions for invoking the enhanced e-commerce capabilities.
+
+### Checkout Flow
+The checkout flow is pre-configured to log purchases through Google Analytics' basic e-commerce capabilities. It will submit transaction/purchase amounts along with cart information where it can be further analyzed in Google Analytics under **Conversions > Ecommerce**. The product and cart pages are configured to submit information when a product is viewed, added to cart, or removed from the cart.
+
+### Global Product Attribute
+The Google Analytics handlers use the global product attribute `Tag` and expect the formatted value of `store-name:category` to submit category information to Google Analytics, for example, "vestri:Accessories". If this global attribute is not set for each product in the catalog, an empty value is submitted to Google Analytics as the product's category.
+
+
```

Comment:
will review these section in the next round once you incorporate generic comments that apply to the entire section